src/game/scenes/arena.c

```diff
@@ -595,12 +604,16 @@ void arena_har_defeat_hook(int loser_player_id, scene *scene) {
                         (player_loser->pilot->money + player_loser->pilot->winnings) * winnings_multiplier;
                     fight_stats->winnings += (int)(400 * hp_percentage);
                 }
-                player_winner->pilot->rank--;
+                if(!gs->match_settings.sim) {
+                    player_winner->pilot->rank--;
+                }
```

Comment:
This is now adding wins and losses to the pilots in sim mode too.

Reply:
Fixed